### ISSUE TITLE
Use of deprecated constants which will be removed in HA Core 2025.1

### ISSUE THREAD

**Issue by shortibln** [RainerStaude/hass-becker-component-plus-pybecker#32]
Hi,

first of all thanks for the great work - the integration works perfectly for me! 
Looking at the logs I saw, that some constants got deprecated. Its still some time, but do you see a chance to adapt the code?

Many thanks upfront. 
Best regards
Michael
 
```
2024-01-05 22:24:41.820 WARNING (MainThread) [homeassistant.components.cover] SUPPORT_CLOSE was used from becker, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.CLOSE instead, please report it to the author of the 'becker' custom integration
2024-01-05 22:24:41.829 WARNING (MainThread) [homeassistant.components.cover] SUPPORT_CLOSE_TILT was used from becker, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.CLOSE_TILT instead, please report it to the author of the 'becker' custom integration
2024-01-05 22:24:41.838 WARNING (MainThread) [homeassistant.components.cover] SUPPORT_OPEN was used from becker, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.OPEN instead, please report it to the author of the 'becker' custom integration
2024-01-05 22:24:41.848 WARNING (MainThread) [homeassistant.components.cover] SUPPORT_OPEN_TILT was used from becker, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.OPEN_TILT instead, please report it to the author of the 'becker' custom integration
2024-01-05 22:24:41.860 WARNING (MainThread) [homeassistant.components.cover] SUPPORT_STOP was used from becker, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.STOP instead, please report it to the author of the 'becker' custom integration
2024-01-05 22:24:41.874 WARNING (MainThread) [homeassistant.components.cover] SUPPORT_SET_POSITION was used from becker, this is a deprecated constant which will be removed in HA Core 2025.1. Use CoverEntityFeature.SET_POSITION instead, please report it to the author of the 'becker' custom integration
```

**Comment by jakubrybar** [comment]
Hi, I am facing the same issue and is now blocking me to use the integration. Did anyone find a solution to it? Is anyone working on this?

**Comment by RainerStaude** [comment]
fixed with version v0.3.4
